junit5/src/main/java/org/jboss/weld/junit5/WeldJunit5Extension.java

```diff
@@ -105,14 +107,14 @@ private static void storeExplicitParamResolutionInformation(ExtensionContext ec)
     }
 
     @Override
-    public void afterAll(ExtensionContext context) throws Exception {
```

Comment:
For the record, removal of the `throws` clause breaks source compatibility for people who extend this class and override these methods.

Reply:
Hmm, that's good point.
I think it's fine with version 4 since it went into 4.0.0.Final which was a major bump but the version 3 shouldn't have done this. Which one are you using?

Reply:
This was backported to 3.1.0 in #134, which is what I'm using. I don't mind too much, I have a "sneaky throw" util around anyway so I could deal with it easily.

I can see how I'd myself do this mistake. My IDE suggests to remove a `throws` clause if the method body doesn't throw anything, and you just don't notice immediately that it would be problematic. Plus I'm really not sure if this class is even meant to be subclassed, so I might be the only one doing it.